libnest/src/config.rs

```diff
@@ -59,10 +63,26 @@ impl Config {
     /// ```
     #[inline]
     pub fn new() -> Config {
-        Config {
-            cache: PathBuf::from("/var/lib/nest/cache/"),
-            download_path: PathBuf::from("/var/lib/nest/download/"),
-            repositories: Vec::new(),
+        Config::default()
+    }
+
+    /// Loads a configuration from a TOML file
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # extern crate libnest;
+    /// use libnest::config::Config;
+    ///
+    /// let mut config = Config::new();
+    /// config.load_conf("Config.toml");
+    /// ```
+    pub fn load_conf(&mut self, path: &str) {
```

Comment:
You should pick an idiomatic Rust name like `from`, `from_path` or `load`.

A more Rust-idiomatic behavior would be that the constructor of Config loads the configuration file and creates a Config with these values.

You should also take a `&Path` instead of an `&str`. Use std types when possible.

Eg:

```rust
iml Config {
    pub fn new() -> Config {
        Config {
            /* Default values */
        }
    }

    pub fn load(path: &Path) -> Result<Config, WhateverErrors> {
        /* Load TOML file and return errors to the caller */
        Ok(Config {
            /* set content here */
        })
    }
}
```